SD2019StackOverflowApp/src/main/java/ro/utcn/sd/mca/SD2019StackOverflowApp/entity/AnswerVote.java

```diff
@@ -0,0 +1,60 @@
+package ro.utcn.sd.mca.SD2019StackOverflowApp.entity;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.*;
+import java.util.LinkedHashMap;
+import java.util.Map;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@Entity
+public class AnswerVote implements DatabaseEntity {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Integer answerVoteId;
+    @Column(name = "stack_overflow_user_id")
+    private Integer sOUserId;
+    private Integer answerId;
+    private String voteType;
+
+    @Override
+    public Integer getId() {
+        return answerVoteId;
+    }
+
+    @Override
+    public void setId(Integer id) {
+        answerVoteId = id;
+    }
+
+    @Override
+    public String getAssociatedTableName() {
+        return "answer_vote";
+    }
+
+    @Override
+    public Map<String, Object> getColumnFieldValueAssociations() {
```

Comment:
Might be an idea to have some specialized "Mapper" classes to handle these transformations (toMap, update).